pylenium/driver.py

```diff
@@ -609,7 +609,10 @@ def window_size(self) -> dict:
     def maximize_window(self) -> 'Pylenium':
         """ Maximizes the current Window. """
         self.log.info('py.maximize_window() - Maximize window')
-        self.webdriver.maximize_window()
+        try:
+            self.webdriver.maximize_window()
+        except WebDriverException as e:
+            self.log.warning(f'py.maximize_window() - FAILED: {e.msg}')
```

Comment:
Change the message to something less "aggressive" haha. Something like:

```
f'unable to maximize window: {e.msg}'
```

Also, do you think this is a `warning` or an `error`?

Reply:
done!